### PR TITLE
Acquire course data

### PR DESCRIPTION
**Pull request by wilsonia**

### CODE REVIEW

File: catalog_data_acquisition/requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+beautifulsoup4==4.7.1
```
Comment (ianbrown9475) [inline catalog_data_acquisition/requirements.txt:1]: Could you also add `requests`?

File: catalog_data_acquisition/get_courses.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    courses_html = requests.get(url).text
+    soup = BeautifulSoup(courses_html, 'html.parser')
+    for course in soup.find_all(class_='courseblock'):
+        title = course.find(class_='courseblocktitle').get_text().split('.')[0]
+        print(title)
+        block = course.find(class_='courseblockextra')
+        prereqs = ''
+        if block is not None:
+            if block.find('strong') is not None:
+                if block.find('strong').get_text() == 'Prerequisites:':
```
Comment (ianbrown9475) [inline catalog_data_acquisition/get_courses.py:20]: You should be able to replace these nested ifs with `and` statements

**Review by ianbrown9475 — CHANGES_REQUESTED** [review]
Looks good overall and I got it to work successfully. Since we shouldn't need to run the scripts that often, they should be fine as they are. If that ever changes, we should look into using asynchronous requests

**Comment by ianbrown9475** [comment]
Webhook test comment

**Comment by wilsonia** [comment]
Thanks @ianbrown9475 , I've implemented suggestions on the introduce-or-nodes branch. Since that one was branched from this and this branch has gone idle, I'll close this PR to focus on [this one](https://github.com/course-network/network-visualization/pull/6) instead

**Pull request closed without merging**